romana-install/config.yml

```diff
@@ -0,0 +1,36 @@
+---
+# Ensure stack nodes are ready for installation
+- hosts: stack_nodes
+  tasks:
+    - name: Wait for cloud-init to complete
+      wait_for: path="/var/lib/cloud/instance/boot-finished" state=present
+
+# Apply common settings to all nodes prior to installation
+- hosts: stack_nodes
+  roles:
+    - system/ubuntu
+
+# Include stack playbooks. Only the configured stack will be installed.
+- include: config.devstack.yml
```

Comment:
We should be able to apply "when: stack_type == ..." here, so that we don't have to have the "when" in the includes themselves and will never include stuff that won't be needed.